content/en/getting_started/_index.md

```diff
@@ -11,117 +11,10 @@ further_reading:
   text: "Introduction to Datadog"
 ---
 
```

Comment:
This page seems very empty now. It seems like we could take advantage of how many hits this page gets and still have a bit more overview content.

Reply:
Like the 3 pillars + 3 products workflow?

Reply:
If we want something like this, we can probably add it later. Going to approve for now.